nifi-registry-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -14,7 +14,1303 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-NiFi Registry System Administrator's Guide
-=================================
+Apache NiFi Registry System Administrator's Guide
+==========================================
 Apache NiFi Team <dev@nifi.apache.org>
 :homepage: http://nifi.apache.org
+
+System Requirements
+-------------------
+NiFi Registry has the following minimum system requirements:
+
+* Requires Java 8 or newer
+* Supported Operating Systems:
+** Linux
+** Unix
+** Mac OS X
+* Supported Web Browsers:
+** Google Chrome:  Current & (Current - 1)
+** Mozilla FireFox: Current & (Current - 1)
+** Safari:  Current & (Current - 1)
+
+**Note** Under sustained and extremely high throughput the CodeCache settings may need to be tuned to avoid sudden performance loss.  See the <<bootstrap_properties>> section for more information.
+
+How to install and start NiFi Registry
+--------------------------------------
+
+* Linux/Unix/OS X
+** Decompress and untar into desired installation directory
+** Make any desired edits in files found under <installdir>/conf
+** From the <installdir>/bin directory, execute the following commands by typing ./nifi-registry.sh <command>:
+*** start: starts NiFi Registry in the background
+*** stop: stops NiFi Registry that is running in the background
+*** status: provides the current status of NiFi Registry
+*** run: runs NiFi Registry in the foreground and waits for a Ctrl-C to initiate shutdown of NiFi Registry
+*** install: installs NiFi Registry as a service that can then be controlled via
+**** service nifi-registry start
+**** service nifi-regsitry stop
+**** service nifi-registry status
+
+
+When NiFi Registry first starts up, the following files and directories are created:
+
+* flow_storage directory
+* work directory
+* logs directory
+* run directory
+
+See the <<system_properties>> section of this guide for more information about NiFi Registry configuration files.
+
+
+Configuration Best Practices
+----------------------------
+
+NOTE: If you are running on Linux, consider these best practices. Typical Linux defaults are not necessarily well tuned for the needs of an IO intensive application like NiFi Registry.  For all of these areas, your distribution's requirements may vary.  Use these sections as advice, but consult your distribution-specific documentation for how best to achieve these recommendations.
```

Comment:
I'm assuming this was copied from the Apache NiFi Admin Guide -- is Registry really I/O intensive?

Reply:
Yes, just left that section in for now.  Basically, any NiFi doc content that could potentially be included in the Registry doc I left, so that it could be edited/modified by someone who knows the details.  I figured if I deleted it, then it might get lost/forgetten.